gradle.properties

```diff
@@ -3,16 +3,16 @@ org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
 	# check these on https://fabricmc.net/use
-	minecraft_version=1.20.6
-	loader_version=0.15.10
+	minecraft_version=1.21
+	loader_version=0.15.11
 
 # Mod Properties
-	mod_version=1.14.0
+	mod_version=1.15.0
```

Comment:
don't worry about bumping the mod version - I take care of that before publishing the release